app/views/events/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<div>
+  <h5>Events</h5>
+  <table>
+    <thead>
+      <tr>
+        <th>Description</th>
+        <th>Date</th>
+        <th colspan="3"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% events.each do |event| %>
+        <tr>
+          <td><%= event_description(event) %></td>
+          <td><%= event_date(event) %></td>
+          <td><%= link_to 'Show', event %></td>
+        </tr>
+      <% end %>
+    </tbody>
```

Comment:
As I mentioned in the controller, there is no need to show the whole list of events if you are already separating them into past and future. Makes things simpler.